### PR TITLE
kvm,snapshot: note on primary storage dependency

### PR DESCRIPTION
**Pull request by shwstppr**
Address https://github.com/apache/cloudstack/issues/8034

----
📚 Documentation preview 📚: https://cloudstack-documentation--367.org.readthedocs.build/en/367/

### CODE REVIEW

**Review by DaanHoogland — APPROVED** [review]

File: source/adminguide/storage.rst [hunk 1]
```diff
@@ -957,6 +957,9 @@ incremental backups are supported, every N backup is a full backup.
 | Support incremental backup   | No               | Yes              | No  |
 +------------------------------+------------------+------------------+-----+
 
+   .. note::
+      Between versions 4.17.x, 4.18.0 and 4.18.1, KVM volume snapshot backups were not full snapshots and they rely on the snapshots on the primary storage. To prevent any loss of data, care must be taken during revert operation and it must be ensured that the source primary storage snapshot file is present.
+
```
Comment (GutoVeronezi) [inline source/adminguide/storage.rst:962]: @shwstppr, would not it be useful to guide users on how to identify the affected volumes/snapshots, vide https://github.com/apache/cloudstack/issues/8034#issuecomment-1750699975?

Comment (shwstppr) [reply source/adminguide/storage.rst:962]: @GutoVeronezi good idea. Though I'm not sure how to structure it.
Would you mind adding a change in the PR as you must be having better understanding on the process?

**Comment by shwstppr** [comment]
@andrijapanicsb @rajujith can you please review?

File: source/adminguide/storage.rst (hunk 1 above)
Comment (GutoVeronezi) [reply source/adminguide/storage.rst:962]: I missed the last comment; I will put it on my tasks :+1:

Comment (shwstppr) [reply source/adminguide/storage.rst:962]: @GutoVeronezi any update or should we merge this?

**Review by rohityadavcloud — APPROVED** [review]

**Review by harikrishna-patnala — APPROVED** [review]
LGTM

File: source/adminguide/storage.rst (hunk 1 above)
Comment (harikrishna-patnala) [reply source/adminguide/storage.rst:962]: @shwstppr may be we can continue merging this and handle that steps in a different PR. @GutoVeronezi can you please raise a separate PR for that.

**Pull request merged**